tests/Stubs/TestControllers/WelcomeController.php

```diff
@@ -8,6 +8,9 @@
 use Rakutentech\LaravelRequestDocs\Tests\Stubs\TestRequests\WelcomeIndexRequest;
 use Rakutentech\LaravelRequestDocs\Tests\Stubs\TestRequests\WelcomeStoreRequest;
 
+/**
+ * @LRDtags Welcome
```

Comment:
This PR added a lot of features, but the test doesn't cover them at all.

You should add tests and the test should break without your added features

Reply:
Foram criados testes para cobrir todas as novas implementações